analysis/deep2_r23_o32/general.py

```diff
@@ -327,13 +327,20 @@ def run_individual_functions(fitspath, want, dataset='n_Bins', n_split=3, adapti
 
     Parameters
     ----------
-    fitspath -> path to the location of files saved for each run
-    want -> keyword to determine what part of the process needs to be run
-         -> Keywords: binning_and_graphing, stack_mastergrid, zoom, R_cal_temp, line_ratio_plotting
-    adaptive  -> determines if the R23 bins have equal or different number of spectra in them in binning method
-    y_correction -> determines if the smoothed (movingaverage_box1D) version of y is used in zoom_and_gauss_general.py
-    apply_dust -> determines if dust attenuation corrections are applied
-
+    :param fitspath: str. path to the location of files saved for each run
+    :param want: str. keyword to determine what part of the process needs to be run
+          options: binning_and_graphing, stack_mastergrid, zoom, R_cal_temp, line_ratio_plotting
+    :param dataset: str. Define binning method options:
+                    'Grid', 'O32_Grid', 'R23_Grid', 'n_Bins'
+    :param n_split: int. Number of times the log(R23) bins are split.
+           Default: 3.  Only for 'Grid' case
+    :param y_correction: bool. Set to use smoothed (movingaverage_box1D)
+           version for spectrum in zoom_and_gauss_general. Default: False
+    :param adaptive: bool. Set to use adaptive binning, otherwise equal
+           log(R23) bins. Default: True
+    :param apply_dust: bool. Apply dust correction. Default: False
+    :param mask: bool. Mask night sky mask in stackingboth_mastergrid.
+           Default: True
```

Comment:
```suggestion
    :param fitspath: str. Path to the location of files saved for each run
    :param want: str. Keyword to determine what part of the process needs to be run
           Options are: "binning_and_graphing", "stack_mastergrid", "zoom", "R_cal_temp", "line_ratio_plotting"
    :param dataset: str. Define binning method options:
           'Grid', 'O32_Grid', 'R23_Grid', 'n_Bins'
    :param n_split: int. Number of times the log(R23) bins are split.
           Default: 3.  Only for 'Grid' case
    :param y_correction: bool. Set to use smoothed (`movingaverage_box1D`)
           Version for spectrum in zoom_and_gauss_general. Default: False
    :param adaptive: bool. Set to use adaptive binning, otherwise equal
           log(R23) bins. Default: True
    :param apply_dust: bool. Apply dust correction. Default: False
    :param mask: bool. Mask night sky mask in stackingboth_mastergrid.
           Default: True
```

Reply:
fixed manually because some of the corrections applied in other run function